src/HasTags.php

```diff
@@ -73,12 +73,13 @@ public function setTagsAttribute($tags)
     /**
      * @param \Illuminate\Database\Eloquent\Builder $query
      * @param array|\ArrayAccess|\Spatie\Tags\Tag $tags
+     * @param null $locale
      *
      * @return \Illuminate\Database\Eloquent\Builder
      */
-    public function scopeWithAllTags(Builder $query, $tags, string $type = null): Builder
+    public function scopeWithAllTags(Builder $query, $tags, string $type = null, $locale = null): Builder
```

Comment:
You can add a `string` type hint to the `locale` parameter

Reply:
Fixed, thank you